packages/devtools/lib/src/utils.dart

```diff
@@ -279,6 +279,7 @@ class RateLimiter {
   void _performRequest() async {
     try {
       _lastRequestTime = DateTime.now().millisecondsSinceEpoch;
+      _pendingRequest = Completer();
```

Comment:
This class was broken but we didn't notice due to the bug in reporting flutter.frame events.